modular_zubbers/code/modules/hydroponics/botanyplumber.dm

```diff
@@ -0,0 +1,3 @@
+/obj/machinery/vending/hydronutrients/New(...)
+    contraband[/obj/item/construction/plumbing] = 2
+    . = ..()
```

Comment:
It's fine to make botany ask chemistry. Same reason why I wouldn't give a round start chem dispensor. Especially when the chemicals people make as a factory are already in the biogen

Reply:
if you say so, just been a long-requested thing. Are the other things in the PR fine?